File: benchmarks/xlang/src/strbuild.py
import time
n = 2000000
t0 = time.perf_counter()
s = ""
for i in range(n):
    s += str(i % 1000)
print(f"CHECK {len(s)}")
print(f"MS {int((time.perf_counter() - t0) * 1000)}")

File: benchmarks/xlang/src/strbuild.go
package main
import ("fmt"; "strconv"; "strings"; "time")
func main() {
    const n = 2000000
    t0 := time.Now()
    var b strings.Builder
    for i := 0; i < n; i++ { b.WriteString(strconv.Itoa(i % 1000)) }
    fmt.Printf("CHECK %d\nMS %d\n", b.Len(), time.Since(t0).Milliseconds())
}
